packages/myst-to-react/src/admonitions.tsx

```diff
@@ -199,7 +199,7 @@ export function Admonition({
           <div
             className={classNames(
               'text-neutral-900 dark:text-white grow self-center overflow-hidden break-words',
-              { 'ml-2': hideIcon },
+              { 'ml-4': hideIcon },
```

Comment:
this gives the proper alignment w/o icon -- compare to https://github.com/executablebooks/myst-theme/blob/abf8f6b02556f7a95bda11e6084391d6a32a3d29/packages/myst-to-react/src/proof.tsx#L164

Reply:
Nice, looks much better. :)

![image](https://github.com/executablebooks/myst-theme/assets/913249/6fef1541-4c2e-4110-bb82-7f1286dfd87b)